packages/sdk-components-react/src/head-slot.tsx

```diff
@@ -26,7 +26,6 @@ export const HeadSlot = forwardRef<
     <div
       ref={ref}
       style={{
-        backgroundColor: "rgba(255,255,255,1)",
```

Comment:
Why remove here?

Reply:
Just felt redundant as XmlNode is going to apply it anyways

Reply:
Ah, got it. Xml node is used by head slot.